lib/rabbit/gtk.rb

```diff
@@ -159,6 +159,22 @@ def new(type)
           end
         end
       end
+
+      alias_method :pack_start_raw, :pack_start
+      def pack_start(child, options={})
+        expand  = options[:expand]  || true
```

Comment:
No. Because Ruby-GNOME2 doesn't use keyword arguments. This is a Ruby-GNOME2 compatible layer. This layer should not use different API.

We can write `pack_start(child, true, true, 0)` if we use keyword argument.